app/tasks/maintenance/create_previews_for_pj_of_latest_dossiers_task.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Maintenance
+  class CreatePreviewsForPjOfLatestDossiersTask < MaintenanceTasks::Task
+    def collection
+      dossier_ids = Dossier
+        .state_en_construction_ou_instruction
+        .where('depose_at > ?', 3.months.ago)
```

Comment:
juste pour ta culture pour être plus ruby-esque :
```suggestion
        .where(depose_at: 3.months.ago..)
```
(`..` sans rien après correspond à un range infini)